automation/id_rsa_nmstate

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Could you please rename this file to be more specific to its usage?
I feel uncomfortable having it, but I see there is no other options at the moment.

Reply:
Done.